package.json

```diff
@@ -36,24 +35,25 @@
     "@segment/analytics-next": "^1.74.0",
     "@vuepress/bundler-vite": "2.0.0-rc.20",
     "@vuepress/helper": "^2.0.0-rc.80",
-    "@vuepress/plugin-docsearch": "^2.0.0-rc.76",
-    "@vuepress/plugin-google-analytics": "^2.0.0-rc.66",
-    "@vuepress/plugin-notice": "^2.0.0-rc.74",
-    "@vuepress/plugin-seo": "^2.0.0-rc.74",
-    "@vuepress/plugin-sitemap": "^2.0.0-rc.74",
+    "@vuepress/plugin-docsearch": "^2.0.0-rc.86",
+    "@vuepress/plugin-notice": "^2.0.0-rc.86",
+    "@vuepress/plugin-seo": "^2.0.0-rc.86",
+    "@vuepress/plugin-sitemap": "^2.0.0-rc.86",
+    "@vuepress/plugin-watermark": "^2.0.0-rc.86",
     "@vueuse/core": "^12.8.2",
     "bowser": "^2.11.0",
     "dashjs": "^4.7.4",
     "hls.js": "^1.5.14",
     "iconify-icon": "^2.1.0",
     "mermaid": "^11.4.0",
-    "sass-loader": "^15.0.0",
+    "posthog-js": "^1.232.7",
+    "sass-loader": "^16.0.5",
     "uglify-js": "^3.19.3",
-    "vue": "^3.4.30",
+    "vue": "^3.5.13",
```

Comment:
do we require the update on vue to implement this change ?

Reply:
if you try running the build with the updated Hope, you will get unmatched references warning. If Hope has a dependency on the Vue version, should we not upgrade?

Reply:
I would add the upgrade as a separate PR, so if we need to rollback we have a different commit to point to. But it's not a big deal